wiki/People/Administrators/en.md

```diff
@@ -0,0 +1,26 @@
+# Titanic Administators
+
+The people who manage and moderate Titanic 
```

Comment:
```suggestion
# Staff

The people who manage, moderate and improve Titanic every day.
```

Reply:
What about something like this?

Reply:
yeah that seems like it works better than what I had put, I only had that there because I was going based on old wiki xd
(also how tf did I miss the internal spelling error as well what)